plugins/finder/jevents/jevents.php

```diff
@@ -66,52 +66,8 @@ class plgFinderJEvents extends FinderIndexerAdapter
 	 * @var    string
 	 * @since  2.5
 	 */
-	protected $table = '#__jevents_vevdetail';
-
-	/**
-	 * Constructor
-	 *
-	 * @param   object  &$subject  The object to observe
-	 * @param   array   $config    An array that holds the plugin configuration
-	 *
-	 * @since   2.5
-	 */
-	public function __construct(&$subject, $config)
-	{
-		parent::__construct($subject, $config);
-		$this->loadLanguage();
-	}
-
-	/*
-	public function onStoreCustomDetails($evdetail)
-	{
-		// Only use this method when editing and saving a specific repeat
-		if (JRequest::getCmd("task")!="icalrepeat.save" && JRequest::getCmd("task")!="icalrepeat.apply" ){
-			return true;
-		}
-		$detailid = $evdetail->evdet_id;
-
-		// Reindex the item
-		$this->reindex($detailid);
-		return true;
-	}
-	 */
-
-	public function onAfterSaveEvent (&$vevent, $dryrun) {
-		if ($dryrun || !isset($vevent->detail_id) || $vevent->detail_id==0){
```

Comment:
We need this - UNLESS we replicate the same code to handle the event detail_id in the content plugin.

We can't switch from detail_id to event id since it won't pick up customised repeats.